std/wasi/snapshot_preview1_test.ts

```diff
@@ -64,7 +64,7 @@ for (const pathname of tests) {
     ignore: ignore.includes(pathname),
     fn: async function () {
       const prelude = await Deno.readTextFile(
-        path.resolve(rootdir, pathname.replace(/\.wasm$/, ".json")),
+        path.resolve(Deno.cwd(), rootdir, pathname.replace(/\.wasm$/, ".json")),
```

Comment:
This probably isn't necessary - rootdir is already absolute, right?

Reply:
Yes, those were included as a quick way to test path.resolve implementations. The same goes for test files which use 
```
const moduleDir = path.dirname(path.fromFileUrl(import.meta.url));
const testdataDir = path.resolve(Deno.cwd(), moduleDir, "testdata");
```
I've removed the unnecessary calls in the latest commit. Thanks for reviewing:)